web-stories.php

```diff
@@ -47,6 +47,7 @@
 define( 'WEBSTORIES_PLUGIN_DIR_URL', plugin_dir_url( WEBSTORIES_PLUGIN_FILE ) );
 define( 'WEBSTORIES_ASSETS_URL', WEBSTORIES_PLUGIN_DIR_URL . 'assets' );
 define( 'WEBSTORIES_MINIMUM_PHP_VERSION', '5.6.0' );
+define( 'WEBSTORIES_MINIMUM_WP_VERSION', '5.3.0' );
```

Comment:
Needs to be added to `tests/phpstan/bootstrap.php` as well

Reply:
Done

Reply:
Fixed